OSDOCS#11332: Removed unnecessary steps for vSphere Problem Detector Operator check (#84711)
openshift/openshift-docs: modules/vsphere-problem-detector-running.adoc

You can override the schedule for running the {operator-name} checks and run the checks immediately.

> The {operator-name} automatically runs the checks every 8 hours. However, when the Operator starts, it runs the checks immediately. The Operator is started by the Cluster Storage Operator when the Cluster Storage Operator starts and determines that the cluster is running on vSphere.

Reviewer: ```suggestion
The {operator-name} automatically runs the checks every 1 hours. However, when the Operator starts, it runs the checks immediately. The Operator is started by the Cluster Storage Operator when the Cluster Storage Operator starts and determines that the cluster is running on vSphere.
```
Ignore the comment if you have updated in another PR.

Author: This is updated in the other PR.

.Procedure

* Scale the Operator to `0`:
+
[source,terminal]
----
$ oc scale deployment/vsphere-problem-detector-operator --replicas=0 \
    -n openshift-cluster-storage-operator
----

.Verification

> . Verify that the pods have restarted by running the following command:

Reviewer 2: This should be unnumbered since it's a single step

```suggestion
* Verify that the pods have restarted by running the following command:
```

.Procedure

* Scale the Operator to `0`:
+
[source,terminal]
----
$ oc scale deployment/vsphere-problem-detector-operator --replicas=0 \
    -n openshift-cluster-storage-operator
----

.Verification

. Verify that the pods have restarted by running the following command:
+
[source,terminal]
----
$ oc -n openshift-cluster-storage-operator get pod -l name=vsphere-problem-detector-operator
----
+
.Example output
|===
|NAME
|READY
|STATUS
|RESTARTS
|AGE

|vsphere-problem-detector-operator-77486bd645-9ntpb
|1/1
|Running
|0
|11s
|===

> 

Reviewer: @snarayan-redhat Hi, the output of the command doesn't good, it should be as below, but the docs preview gives a table, could you check please?
```
NAME                                                 READY   STATUS    RESTARTS   AGE
vsphere-problem-detector-operator-77486bd645-9ntpb   1/1     Running   0          11s
```

Author: Thanks for the catch. It seems to look fine now. Can you give another look?

> . View the events or logs that are generated by the {operator-name}. Confirm that the events or logs have recent timestamps.

Reviewer: ```suggestion
. View the "AGE" field of the {operator-name} from the output above. Confirm that is new generated.
```

    --for=delete --timeout=5m -n openshift-cluster-storage-operator

> ----

Reviewer: It's better to remove this part, as the vsphere-problem-detector-operator is managed, the pod will be scaled very quickly and you will probably wait till timeout.
Suggest to use alternative way:

The vsphere-problem-detector-operator will be scaled up automatically, so you can run the following command to check the AGE of pod to see if it is restarted:
```
oc -n openshift-cluster-storage-operator get pod -l name=vsphere-problem-detector-operator

NAME                                                 READY   STATUS    RESTARTS   AGE
vsphere-problem-detector-operator-77486bd645-9ntpb   1/1     Running   0          11s
```

Author: oh I see. Is reducing the timeout an option? But I do see why the alternative you provided is more efficient.

Author: How about having both commands?

Reviewer: The pod will start up very very quickly, so you probably have no chance to see the deletion of the old one, you will already get a new one when you execute the "oc wait" cmd and it doesn't work (just timeout). 
If you want to continuously check the status, we can add "w" in my command, I don't think the previous cmd works.
```
oc -n openshift-cluster-storage-operator get pod -l name=vsphere-problem-detector-operator -w

NAME                                                 READY   STATUS    RESTARTS   AGE
vsphere-problem-detector-operator-77486bd645-9ntpb   1/1     Running   0          11s
```

Author: @duanwei33 I have incorporated your suggestions. Can you review if it looks fine?
